tests/TestClassFormModelBuilderDBEstateListSettings.php

```diff
@@ -215,4 +216,29 @@ public function testCreateInputModelShowReferenceEstate()
 		$this->assertEquals($pInputModelDB->getValue(), '0');
 		$this->assertEquals('checkbox', $pInputModelDB->getHtmlType());
 	}
+
+	public function testCreateInputModelEmbedCode()
+	{
+		$pInstance = $this->getMockBuilder(FormModelBuilderDBEstateListSettings::class)
+			->disableOriginalConstructor()
+			->setMethods(['getInputModelDBFactory', 'getValue'])
+			->getMock();
+
+		$pInstance->method('getInputModelDBFactory')->willReturn($this->_pInputModelFactoryDBEntry);
+		$pInstance->method('getValue')->willReturn('1');
+		$pInputModelFormEmbedCode = $pInstance->createInputModelEmbedCode();
+		$this->assertInstanceOf(InputModelLabel::class, $pInputModelFormEmbedCode);
+		$this->assertEquals($pInputModelFormEmbedCode->getHtmlType(), 'label');
+	}
+
+	public function testCreateInputModelButton()
+	{
+		$pInstance = $this->getMockBuilder(FormModelBuilderDBEstateListSettings::class)
```

Comment:
Testing a class using a mock of itself doesn't really make sense

Reply:
i fixed it, please check PR help me !